node/src/components/chain_synchronizer/operations.rs

```diff
@@ -1339,10 +1339,11 @@ impl BlockSignaturesCollector {
     ) -> bool {
         self.0.as_ref().map_or(false, |sigs| {
             are_signatures_sufficient_for_sync_to_genesis(
-                consensus::check_sufficient_finality_signatures(
```

Comment:
This is only used when syncing back to genesis, right? For syncing forward we request headers with _(1+f)/2_ signatures?

Reply:
Yes, that's true. Only in `sync_to_genesis` we use `check_sufficient_finality_signatures_with_quorum_formula`. Other places use the classic variant `check_sufficient_finality_signatures` which requires the (1+f)/2 by default.